### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by bensoltoff** [uc-dataviz/critique-rodrigovaldes#1]
## Viz critique

* I would think of this graph more appropriate in exploratory data analysis. Quick to produce, but needs more work for publication. Though given its source as a practice graph for teaching data science it would appear to meet the needs
* Lacks a legend for the point size. What is this even telling us?

## Grammar of graphics

* Good data, and good defense of your design
* I don't fully buy into all of Tufte's arguments, but to publish this graph somewhere I would switch to a more minimalistic theme. At least one without the gray background
* Also in combining the two plots, it would be better if the x-axes lined up perfectly with one another. Hard to do, but see the example of Minard's graph. It shows exactly how to do this and I think it makes a substantial impact on the final product

**Comment by rodrigovaldes** [comment]
I agree, the problem with this graph is that it can be misleading, especially for the non-trained eye. 

After your Tufte defense during the last class, I will try to be closer to his recommendations. Previous versions of the graph had x-axes that lined up perfectly. However, I believe it changed when I added the labels for the y-axes. I'll consider those details for the next assignment.
